src/libs/Notification/PushNotification/subscribeToReportCommentPushNotifications.ts

```diff
@@ -30,6 +31,10 @@ Onyx.connect({
  */
 export default function subscribeToReportCommentPushNotifications() {
     PushNotification.onReceived(PushNotification.TYPE.REPORT_COMMENT, ({reportID, reportActionID, onyxData, lastUpdateID, previousUpdateID}) => {
+        if (!ActiveClientManager.isClientTheLeader()) {
```

Comment:
Just an FYI, mobile is always the leader (there's only one client) so we'll never hit this code block

Reply:
Ok. This is not creating a problem, right?

Reply:
Nope just noticed this while refactoring. I'll probably remove it.